frontend/src/state/columns.js

```diff
@@ -0,0 +1,28 @@
+import { readable } from 'svelte/store';
+
+const breakPointLarge = 900;
+const breakPointMiddle = 768;
+const breakPointSmall = 512;
+
+const columnStore = readable(7, (set) => {
+  let resizeCallback = () => {
+    let width = window.innerWidth;
+
+    if (width > breakPointLarge) {
+      set(7);
+    } else if (width > breakPointMiddle) {
+      set(5);
+    } else if (width > breakPointSmall) {
+      set(4);
+    } else {
+      set(3);
+    }
+  };
+
+  resizeCallback();
+  window.addEventListener('resize', resizeCallback);
```

Comment:
I like the usage of a readable store as the place for the resize handling!

not sure about the columns calculation though, shouldn't this be managed via the media queries that already exist?
also breakpoint defintions are duplicated that way.